r/configure

```diff
@@ -153,8 +153,7 @@ else
       fi
 
       if [ "${ARROW_DEPENDENCY_SOURCE}" = "" ]; then
-        # TODO: BUNDLED is still default for now, but we plan to change it to AUTO
-        ARROW_DEPENDENCY_SOURCE=BUNDLED; export ARROW_DEPENDENCY_SOURCE
+        ARROW_DEPENDENCY_SOURCE=AUTO; export ARROW_DEPENDENCY_SOURCE
```

Comment:
(Can't leave this comment below but) on L162, we should revise the message, maybe just

```
echo "**** pkg-config not installed, setting ARROW_DEPENDENCY_SOURCE=BUNDLED"
```

Reply:
Done!